osidb_bindings/bindings/pyproject.toml

```diff
@@ -1,39 +1,27 @@
 [tool.poetry]
 name = "bindings"
-version = "4.6.0"
+version = "4.6.1"
 description = "A client library for accessing OSIDB API"
-
 authors = []
-
 readme = "README.md"
 packages = [
     {include = "python_client"},
 ]
 include = ["CHANGELOG.md", "python_client/py.typed"]
 
+
 [tool.poetry.dependencies]
-python = "^3.6"
-httpx = ">=0.15.4,<0.21.0"
-attrs = ">=20.1.0,<22.0.0"
+python = "^3.9"
+httpx = ">=0.20.0,<0.28.0"
+attrs = ">=21.3.0"
 python-dateutil = "^2.8.0"
 
 [build-system]
-requires = ["poetry>=1.0"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
 
-[tool.black]
+[tool.ruff]
 line-length = 120
```

Comment:
not sure if relevant but there's an inconsistency with ruff config https://github.com/RedHatProductSecurity/osidb-bindings/blob/4f6eca5be60601ddb95935d05391b44317129768/ruff.toml#L32

Reply:
I will probably change that in the pyproject.toml, but it is probably not much relevant, the pyproject.toml there is simply generated to introduce the generated client, but I just use the code in the abstraction layer and not introduce it as a standalone package, so I just don't use the pyproject.toml at all. I can probably check the pyproject.toml template and either don't generate it, or provide just some minimal info and defer rest to the top level of the package. Wdyt?

Reply:
They have a `pyproject_ruff` template that we could include and copy the config from the top level `ruff.toml` https://github.com/openapi-generators/openapi-python-client/blob/main/openapi_python_client/templates/pyproject_ruff.toml.jinja

We don't expose the `bindings` project outside of the `osidb_bindings` so maybe can be removed safely, one less file that we need to update on each release